react/TextField/TextField.js

```diff
@@ -88,11 +88,12 @@ export default class TextField extends Component {
   }
 
   renderInput() {
-    const { inputProps = {}, id } = this.props;
+    const { inputProps = {}, id, valid } = this.props;
     const { ref } = inputProps;
     const allInputProps = {
       ...combineClassNames(inputProps, styles.input),
       ...(id ? { id } : {}),
+      ...{ 'aria-invalid': valid === false },
```

Comment:
My thinking on this was that you probably wouldn't want to set `aria-invalid` to be out of sync with `valid`.  Can you think of a use case where you would need to set `aria-invalid` to something custom?